packages/core/src/intl/languages.test.ts

```diff
@@ -1,3 +1,5 @@
+import { expect } from '@playwright/test';
```

Comment:
Seems like this import was made by mistake, this is a jest test, so `expect` should be in the global context

Reply:
8beebd995dfbe32f8bbb81078e33e5ceb1d97968